test/test_results.py

```diff
@@ -0,0 +1,158 @@
+import unittest
+import sys
+import os
+import subprocess
+import warnings
+
+# Add the parent directory to the Python path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from test.utils.custom_tsv_parser import ParseTSVFile
+
+PREF_CUTOFF = -4
+
+
+def get_scode_cscode_id(data):
```

Comment:
Is this function just for this test or could have been used in the actual code ?

Reply:
Could we make it private or mark it as the fixture with pytest ?